src/layman/map/client_test.py

```diff
@@ -13,7 +13,7 @@
 from layman.map import MAP_TYPE
 from layman.uuid import check_redis_consistency
 
-num_maps_before_test = 0
+NUM_MAPS_BEFORE_TEST = 0
```

Comment:
As it is variable and not constant, I prefer lower case and `# pylint: disable=invalid-name`